docs/en/rst/integrating/auth0.rst

```diff
@@ -5,18 +5,20 @@ Adding an Auth0 Custom Social Integration
 
 Bugzilla can be added as a 'Custom Social Connection'.
 
-====================  ============================================      ======================================================
+====================  ===============================================   ======================================================
 Parameter             Example(s)                                        Notes
---------------------  --------------------------------------------      ------------------------------------------------------
-Name                  BMO-Stage                                         Whatever makes you happy
+====================  ===============================================   ======================================================
+Name                  BMO=Stage                                         Whatever makes you happy
```

Comment:
When getting the docs to build locally using sphinx, I noticed the warning that the above docs had issues so I went ahead and fixed them so that the warnings would go away. I can put this in another PR if you like.

Reply:
Even if they were in a different PR I would expect it should stay as `BMO-Stage` instead of becoming `BMO=Stage` right?

Reply:
Ah I see what you meant. I thought you were asking why I updated the file itself for this PR. I am not sure why my editor changed that character but I can definitely put it back.